src/Components/Scenes/Hydration/JobDesigner/DiagramApp/TransNode/TransNodeModel.js

```diff
@@ -0,0 +1,26 @@
+import { NodeModel } from '@projectstorm/react-diagrams';
+import * as _ from 'lodash';
```

Comment:
[This](https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark/) came up in my recent branch - it seems like you're only using a couple things from `lodash`, so we might be able to reduce bundle size by a lot if we don't anticipate using more

Reply:
Great idea! This was definitely copy/paste leftover from some of the diagram demo code I grabbed. Will change this import to be less wild-cardy.